docs/generated/oadm_by_example_content.adoc

```diff
@@ -201,6 +201,38 @@ Manage nodes - list pods, evacuate, or mark ready
 ====
 
 
+== oadm pod-network join-projects
+Join project network
+
+====
+
+[options="nowrap"]
+----
+	# Allow project p2 to use project p1 network
```

Comment:
I fixed in the code yesterday to use # instead of // (https://github.com/openshift/openshift-sdn/pull/180) but I couldn't pull that change from openshift-sdn repo as there were other commits on top of this bump version that are not compatible with origin repo.
Okay, I will fix this one as a separate issue.